neo/NeoSystem.cs

```diff
@@ -30,10 +30,11 @@ public class NeoSystem : IDisposable
         private ChannelsConfig start_message = null;
         private bool suspend = false;
 
-        public NeoSystem(Store store)
+        public NeoSystem(string storageEngine = null)
         {
-            this.store = store;
             Plugin.LoadPlugins(this);
+            //TODO: Create MemoryStore by default
```

Comment:
@erikzhang Are you working on it? if do you want, i can do it

Reply:
The difficulty with `MemoryStore` is that it is difficult to implement `Find()` in a simple way. I still need to think about how to choose.

Reply:
I think that `MemoryStore` is for developers, so speed is not a problem for me.

Reply:
@erikzhang Do you want me to implement `MemoryStore`? I think that we can optimize `Find` later.

Reply:
I think this would be a good decision, this way we can start detaching the storage. It will help us in the RPC discussion too.

Reply:
Also, the memory will be the default one, right? Does it mean it will be outside of the plugin? When "storageEngine is null", we use the MemoryStore?

Reply:
Yes, the idea is memory by default.

Reply:
OK. I will do it today.

Reply:
neo-express uses in-memory storage for running checkpoints. The checkpoint store for Neo 2 is available here: https://github.com/neo-project/neo-express/blob/master/src/neo2/Persistence/CheckpointStore.cs